react/src/components/molecules/EmergencyAlert/EmergencyAlert.stories.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withKnobs, text, select } from '@storybook/addon-knobs';
+import EmergencyAlert from './index';
+import Link from '../Link';
+
+storiesOf('molecules', module)
+  .addDecorator(withKnobs({ escapeHTML: false }))
+  .add(
+    'EmergencyAlert', (() => {
+      const props = {
+        theme: select('theme', {
+          'c-warning (default)': 'c-warning',
+          'no theme': '',
+          'c-primary': 'c-primary',
+          'c-primary-alt': 'c-primary-alt',
+          'c-gray-dark': 'c-gray-dark'
+        }, 'c-warning'),
+        message: text('message', 'MassPort is operating with limited flights due to road closures.'),
+        timeStamp: text('timeStamp', '02.15.16, 5:00 p.m'),
+        link: ({ theme, linkClasses, TextElement }) => <Link icon={null} classes={linkClasses} href="https://www.mass.gov"><TextElement>Read full alert</TextElement></Link>
```

Comment:
```suggestion
        link: ({ linkClasses, TextElement }) => <Link icon={null} classes={linkClasses} href="https://www.mass.gov"><TextElement>Read full alert</TextElement></Link>
```
theme is not consumed in Link, so don't think we need to pass here. Should also take out from linkArgs?

Reply:
It shouldn't be taken out. This is to give the ability to directly pass the theme to the link function, in case somehow it wasn't known to whatever is using EmergencyAlert.

Reply:
gotcha, ok!